clusterctl/cmd/create_cluster.go

```diff
@@ -95,12 +95,11 @@ func RunCreate(co *CreateOptions) error {
 	d := clusterdeployer.New(
 		mini,
 		clusterdeployer.NewClientFactory(),
-		pd,
 		string(pc),
 		string(ac),
-		co.KubeconfigOutput,
 		co.CleanupExternalCluster)
-	return d.Create(c, m, pcsFactory)
+	err = d.Create(c, m, pd, co.KubeconfigOutput, pcsFactory)
```

Comment:
if you aren't going to log this error, just do `return d.Create(...)` like it was before.

Reply:
Whoops, not sure why I did this.